core/node/default_config.yaml

```diff
@@ -22,6 +22,8 @@ baseChain:
     chainId: 31337
     networkUrl: 'http://127.0.0.1:8545'
     blockTimeMs: 2000
+    # TODO: make sure this does not get propagated to production
+    positiveEntitlementCacheTTLSeconds: 5 # Short TTL for testing
```

Comment:
this config is used as default config in the docker image, and this will propagate it to production. 
use config-template.yaml for CI/local runs

Reply:
Please us yq as in the create_multi.sh for setting the correct values for testing.

Reply:
@sergekh2 to be clear, values in config-template do NOT make it into gamma or production?